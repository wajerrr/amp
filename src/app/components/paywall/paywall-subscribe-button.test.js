import React from 'react';
import renderer from 'react-test-renderer';
import PaywallSubscribeButton from './paywall-subscribe-button';

describe('PaywallSubscribeButton', () => {
  it('should match the snapshot', () => {
    const paywallSubscribeButton = renderer
      .create(
        <PaywallSubscribeButton
          data={{
            id: 'regwall-subscribe-button-1',
            type: 'subscribeButton',
            content: {
              href:
                'https://subscription.economist.com/DE/EngCore/Ecom/RegWall',
              text: 'Subscribe',
            },
          }}
        />
      )
      .toJSON();
    expect(paywallSubscribeButton).toMatchSnapshot();
  });
});
