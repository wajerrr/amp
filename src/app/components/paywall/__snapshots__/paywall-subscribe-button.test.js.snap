// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaywallSubscribeButton should match the snapshot 1`] = `
<div
  className={undefined}
>
  <a
    className="e18299l00 css-18wxk9z-StyledLink-LargeSubscribeButton e1ob2h4l0"
    href="https://econ.st/2bs1gXz"
    role="button"
  >
    Subscribe
  </a>
</div>
`;
